### ISSUE TITLE
Hidden focusable elements on dashboard widgets

### ISSUE THREAD

**Issue by missmatsuko** [craftcms/cms#6885]
### Description
<img width="1233" alt="Screenshot of dashboard widget with visually hidden submit button in focus." src="https://user-images.githubusercontent.com/17884198/93940902-28784300-fce2-11ea-8d12-3a5973453917.png">

It seems like every (or most?) dashboard widget has a visually hidden submit and cancel buttons.

Focus can move to these elements, but users can't see the elements that are in focus.

It doesn't seem like users should be able to access these buttons.

If these elements must be in the DOM for some programmatic function, they should have `tabindex='-0'` to remove it from the tab order.

### Steps to reproduce
1. Use keyboard to tab through interactable elements.

### Additional info

- Craft version: 3.5.11.1

**Comment by brandonkelly** [comment]
Fixed for the next release! Turns out the flip side of the widgets were only getting `opacity: 0` but not `display: none`.

**Comment by brandonkelly** [comment]
Craft 3.5.12 is out now with that fix.
